src/NodaTime/ZonedDateTime.cs

```diff
@@ -378,6 +378,7 @@ public override int GetHashCode()
         /// <param name="lhs">First operand of the comparison</param>
         /// <param name="rhs">Second operand of the comparison</param>
         /// <returns>true if the <paramref name="lhs"/> is strictly earlier than <paramref name="rhs"/>, false otherwise.</returns>
+        [Obsolete("Use ZonedDateTime.Comparer.Instant to compare values by instant for compatibility with 2.0, and compare calendars and zones explicitly.")]
```

Comment:
Is this "Use ... to compare values by instance [...], and [to] compare calendars and zones explicitly", or "Use ... to compare values by instance [...], and [you should] compare calendars and zones explicitly"?